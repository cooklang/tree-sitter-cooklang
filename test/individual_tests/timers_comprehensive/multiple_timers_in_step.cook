Cook for ~{5%minutes}, then simmer ~{20%minutes}