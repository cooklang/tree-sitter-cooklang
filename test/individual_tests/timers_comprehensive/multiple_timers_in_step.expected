(recipe
  (step
    (plain_text)
    (timer
      (amount
        (quantity)
        (units)))
    (plain_text)
    (timer
      (amount
        (quantity)
        (units)))))